clients/gtest/herk_gtest.yaml

```diff
@@ -0,0 +1,117 @@
+---
+include: rocblas_common.yaml
+include: known_bugs.yaml
+
+Definitions:
+  - &quick_matrix_size_range
+    - { N:  -1,  lda:  -1,  K:  1,  ldc:  1 } # bad n
+    - { N:   0,  lda:   2,  K: -1,  ldc: -1 } # bad k
+    - { N:   0,  lda:   3,  K:  3,  ldc:  3 } # n==0
+    - { N:   3,  lda:   3,  K:  0,  ldc:  3 } # k==0
+    - { N:   3,  lda:   1,  K:  3,  ldc:  3 } # bad lda
+    - { N:   3,  lda:   3,  K:  3,  ldc:  1 } # bad ldc
```

Comment:
Might want to add tests where N > lda > K, or N < lda < K,  since these cases are ok (depending on transpose).